packages/headless/src/api/search/search/product.ts

```diff
@@ -1,15 +1,82 @@
 export type Product = {
+  /**
+   * The SKU of the product.
+   */
   sku: string;
+  /**
+   * Name of the product.
+   *
+   * From the `ec_name` field.
+   */
   name: string;
-  thumbnailUrl: string;
+  /**
+   * Description of the product.
+   *
+   * From the `ec_description` field.
+   */
+  description: string;
+  /**
+   * A direct link to the product in URL format.
+   */
```

Comment:
I the context of a "product recommendation", I don't think we will want to render the `description` :thinking: 

@kpoulincoveo should we *not* expose the description here?

Reply:
Should we do the same for `shortDescription`?